src/tests/pages/ModelViewPage.test.tsx

```diff
@@ -1,6 +1,7 @@
 import ModelViewPage from '@/pages/ModelViewPage'
 import { ModalProvider } from '@/providers/ModalProvider'
 import { initialState, default as model, setCoorinatesMatrix, setIndicesMatrix } from '@/redux/slices/modelSlice'
+import modelViewSettingReducer from '@/redux/slices/modelViewSettingSlice.ts'
```

Comment:
the `.ts` file extension is unnecessary
```suggestion
import modelViewSettingReducer from '@/redux/slices/modelViewSettingSlice'
```